src/dropdown.stories.ts

```diff
@@ -157,6 +158,18 @@ async (query: string): Promise<GlideCoreDropdownOption[]> => {
         type: { summary: 'boolean' },
       },
     },
+    loading: {
+      table: {
+        defaultValue: { summary: 'false' },
+        type: {
+          summary: 'boolean',
+          detail: `
+// Add this attribute when asychronously updating Dropdown's default slot. Remove it after the
```

Comment:
```suggestion
// Add this attribute when asynchronously updating Dropdown's default slot. Remove it after the
```

Reply:
Thanks. Trying to meet my typo quota for the day.